src/master-playlist-controller.js

```diff
@@ -846,39 +866,48 @@ export class MasterPlaylistController extends videojs.EventTarget {
    * @return {TimeRange} the seekable range
    */
   seekable() {
+    if (!this.seekable_) {
```

Comment:
I want to say I had a reason for doing this, but I can't recall what it would've been. Looking at it now don't really see any harm in setting `this.seekable_ = videojs.createTimeRanges()` at the start. I'll give it a shot and make sure it doesn't break things.